tests/tools/tnimscriptwithmacro.nims

```diff
@@ -0,0 +1,22 @@
+discard """
+cmd: "nim e $file"
```

Comment:
Are you sure this works? I think the tester runs ``t*.nim`` tests, not ``t*.nims`` tests.

Reply:
good catch. Do you have any suggestion? Should I update the pattern to ``t*.nim[s]``, or should I rename the file to not have a trailing s?

Reply:
Renaming it to ``.nim`` should be fine.